src/NUnitEngine/Addins/nunit.v2.driver.tests/PairwiseTests.cs

```diff
@@ -8,7 +8,7 @@
 using System.Collections;
 using NUnit.Framework;
 
-namespace NUnit.Core.Tests
+namespace NUnit.Engine.Drivers.Tests
```

Comment:
Then change its directory to match the namespace.

On Saturday, April 11, 2015, CharliePoole <notifications@github.com> wrote:

> In src/NUnitEngine/Addins/nunit.v2.driver.tests/PairwiseTests.cs
> <https://github.com/nunit/nunit/pull/586#discussion_r28158449>:
>
> > @@ -8,7 +8,7 @@
> >  using System.Collections;
> >  using NUnit.Framework;
> >
> > -namespace NUnit.Core.Tests
> > +namespace NUnit.Engine.Drivers.Tests
>
> Again... these are tests of the NUnit 2.6.4 core, not of the driver.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nunit/nunit/pull/586/files#r28158449>.
>